packages/sdk/dappkit/README.md

```diff
@@ -103,19 +103,150 @@ requestTxSig(
 );
 
 const dappkitResponse = await waitForSignedTxs(requestId);
-const tx = dappkitResponse.rawTxs[0];
-
-// Send the signed transaction via web3
-kit.web3.eth.sendSignedTransaction(tx).on("confirmation", async () => {
-  const [cUSDBalanceBig, cUSDDecimals] = await Promise.all([
-    stableToken.balanceOf(this.state.address),
-    stableToken.decimals()
-  ]);
-  const cUSDBalance = this.convertToContractDecimals(
-    cUSDBalanceBig,
-    cUSDDecimals
-  );
-
-  this.setState({ cUSDBalance, isLoadingBalance: false });
-})
+const rawTx = dappkitResponse.rawTxs[0];
+
+// Send the signed transaction via the kit
+const tx = kit.connection.sendSignedTransaction(rawTx);
+
+const receipt = await tx.waitReceipt();
+
+const [cUSDBalanceBig, cUSDDecimals] = await Promise.all([
+  stableToken.balanceOf(this.state.address),
+  stableToken.decimals()
+]);
+const cUSDBalance = this.convertToContractDecimals(
+  cUSDBalanceBig,
+  cUSDDecimals
+);
+
+this.setState({ cUSDBalance, isLoadingBalance: false });
+```
+
+## ContractKit example
+
+Let's make a common use example. What about exchanging some cUSD to CELO, and
+then Lock that CELO to be able to vote a validator group?
+Let's do that
+
+([expo base template commit](https://github.com/celo-org/dappkit-base/commit/cf35c82d7650e7b6bc7208ece32440d3a32d9cc5))
+
+```javascript
+import {
+  requestTxSig,
+  waitForSignedTxs
+} from "@celo/dappkit";
+
+// Let's assume that the address has funds enough in cUSD to pay the
+// transactions fees of all the transactions and enough to buy 10 CELOs
+// AND it's already a registered Account (otherwise it will require to call
+// the `createAccount` method from the Accounts contract)
```

Comment:
nit:
```suggestion
// Let's assume that the address has enough cUSD to pay the
// transaction fees of all the transactions and enough to buy 10 CELO
// AND it's already a registered Account (otherwise it will require a call
// to the `createAccount` method from the Accounts contract)
```